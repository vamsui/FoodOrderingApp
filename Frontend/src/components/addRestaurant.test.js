import React from 'react';
import { render, fireEvent, screen, act } from '@testing-library/react';
import Addrestaurant from './addRestaurant';

import {createRenderer} from 'react-test-renderer/shallow'

import { MemoryRouter } from "react-router-dom";



const renderer=createRenderer()

describe("Add Restaurant page Screenshot test",()=>{
  test('renders correctly the Add restaurant component',()=>{
  renderer.render(
    <MemoryRouter initialEntries={['/']}>
      <Addrestaurant/>
    </MemoryRouter>
  );
  const result=renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
})
