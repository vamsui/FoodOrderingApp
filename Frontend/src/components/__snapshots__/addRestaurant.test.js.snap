// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Restaurant page Screenshot test renders correctly the Add restaurant component 1`] = `
<Router
  location={
    Object {
      "hash": "",
      "key": "default",
      "pathname": "/",
      "search": "",
      "state": null,
    }
  }
  navigationType="POP"
  navigator={
    Object {
      "action": "POP",
      "createHref": [Function],
      "createURL": [Function],
      "encodeLocation": [Function],
      "go": [Function],
      "index": 0,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "default",
        "pathname": "/",
        "search": "",
        "state": null,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Addrestaurant />
</Router>
`;
